app/javascript/tests/public_health/bulk_actions/UpdateCaseStatus.test.js

```diff
@@ -71,7 +71,7 @@ describe('UpdateCaseStatus', () => {
       .forEach((option, index) => {
         expect(option.text()).toEqual(CASE_STATUS_OPTIONS[Number(index)]);
       });
-    expect(wrapper.find(Modal.Body).find('p').at(1).text()).toContain('The selected cases will be moved from the isolation workflow to the exposure workflow and placed in the symptomatic, non-reporting, or asymptomatic line list as appropriate.');
+    expect(wrapper.find(Modal.Body).find('span').at(0).text()).toContain('The selected cases will be moved from the isolation workflow to the exposure workflow and placed in the symptomatic, non-reporting, or asymptomatic line list as appropriate.');
```

Comment:
I don't think the `at(0)` is needed here

Reply:
[08c924c](https://github.com/SaraAlert/SaraAlert/pull/1242/commits/08c924c2f1fabd082e3b7c573f38371ba36d08a8)